src/resolver.js

```
@@ -1,25 +1,40 @@
-import { isFunction } from 'lodash';
+import { isNil, isFunction } from 'lodash';
 import { bind, getContainer } from '@globality/nodule-config';
 
 
-/* Default mask function: just passes args.
+/* Default mask function: preserves standard arguments
  */
-function defaultMask(obj, args) {
-    return [args];
+function defaultMask(obj, args, context, info) {
+    return [obj, args, context, info];
```

Note that the new default mask passes `obj` first, not `args`. This means that most existing code will have to define its own `mask` function or change the signatures of `aggregate` and `transform` functions.